### PR TITLE
Update dependency-injection.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
@david-szabo97 please review

cc @guardrex

### CODE REVIEW

File: aspnetcore/fundamentals/dependency-injection.md [hunk 1]
```diff
@@ -316,7 +316,7 @@ DI is an *alternative* to static/global object access patterns. You may not be a
 
 ## Recommended patterns for multi-tenancy in DI
 
-[Orchard Core](https://github.com/OrchardCMS/OrchardCore) is an application framework for building modular, multi-tenant applications on ASP.NET Core. For more information, see the [Orchard Core Documentation](https://docs.orchardcore.net/en/latest/).
+[Orchard Core](https://github.com/OrchardCMS/OrchardCore) is an application framework for building modular, multi-tenant applications on ASP.NET Core. For more information, see the [Orchard Core Documentation](https://docs.orchardcore.net/).
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/dependency-injection.md:319]:
```suggestion
[Orchard Core](https://github.com/OrchardCMS/OrchardCore) is an application framework for building modular, multi-tenant applications on ASP.NET Core. For more information, see the [Orchard Core Documentation](https://docs.orchardcore.net).
```

**Comment by Rick-Anderson** [comment]
cc @guardrex

**Comment by guardrex** [comment]
I tried the link this way and they did a redirect. I usually place the landing link if it seems stable, but this is probably truly the most stable link of all on your PR.

**Comment by Rick-Anderson** [comment]
Ha, I was going to remove the `/en` without checking. Good thing I checked.

**Comment by david-szabo97** [comment]
LGTM

Although I also prefer the landing link to prevent unnecessary redirects, as suggested by @guardrex. If the current link redirects based on browser language or something,  then it makes sense to use it. Otherwise, I don't see much value in changing it to a link that redirects. Maybe the only reason is long-term stability.

**Comment by guardrex** [comment]
We ***usually*** try to provide links to avoid as many redirects as possible (for example, add/subtract a trailing slash to avoid a redirect based on how a server processes them). These cats may have localization on their roadmap, so Rick's idea is probably best here.

**Pull request merged**